madmom/audio/spectrogram.py

```diff
@@ -1115,14 +1115,15 @@ class SpectrogramDifferenceProcessor(BufferProcessor):
 
     def __init__(self, diff_ratio=DIFF_RATIO, diff_frames=DIFF_FRAMES,
                  diff_max_bins=DIFF_MAX_BINS, positive_diffs=POSITIVE_DIFFS,
-                 stack_diffs=None, **kwargs):
+                 stack_diffs=None, sum_diffs=False, **kwargs):
         # pylint: disable=unused-argument
         super(SpectrogramDifferenceProcessor, self).__init__(diff_frames)
         self.diff_ratio = diff_ratio
         self.diff_frames = diff_frames
         self.diff_max_bins = diff_max_bins
         self.positive_diffs = positive_diffs
         self.stack_diffs = stack_diffs
+        self.sum_diffs = sum_diffs
```

Comment:
This is not the right place to do summing of the differences -- nor is there a need to have an option for this.

You can simply add ``np.sum`` as a processor to the processing chain wherever you wanted to use this option.